docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/aws-lambda/nodejs.md

```diff
@@ -52,6 +52,8 @@ It is very simple to instrument your AWS NodeJS Lambda function using the Sumo L
 
      * `SUMOLOGIC_HTTP_TRACES_ENDPOINT_URL` has to be set to send all gathered telemetry data to Sumo Logic. The URL comes from an [HTTP Traces Endpoint URL](/docs/apm/traces/get-started-transaction-tracing/http-traces-source.md). You can use an existing Source or create a new one if needed.
 
+     * {@import ../../../../../reuse/otel-deployment-environment-tag.md}
```

Comment:
Please put it as a part of OTEL_RESOURCE_ATTRIBUTES. It's not separated environment variable.